tests/publiccloud/aistack_basic.pm

```diff
@@ -0,0 +1,130 @@
+    script_run("sh ./install_rke2.sh");
+    script_run("echo 'export PATH=\$PATH:/opt/rke2/bin' >> ~/.bashrc");
+    systemctl("enable rke2-server.service");
+    systemctl("start rke2-server.service");
+    systemctl("status rke2-server.service");
+    script_run("rke2 --version");
+    script_run("curl -sSL $helm_url -o ./install_helm.sh && chmod 775 ./install_helm.sh");
+    script_run("sh ./install_helm.sh");
+    script_run("helm version");
+    script_run("curl -sSL $kubectl_url -o ./kubectl && chmod +x ./kubectl");
+    script_run("sudo mv ./kubectl /usr/local/bin/");
+    script_run("kubectl version --client");
+}
+
+sub install_aistack_chart {
+    my ($instance, $cert_repo, $ing_repo) = @_;
+    record_info('Install AISTACK helm charts');
+    my $username = $ENV{'USER'};
+    my $curl_options = "-sSL --retry 3 --retry-delay 30";
+    my $APPLICATION_COLLECTION_USER_PASSWORD_SECRET ="";
+    my $docker_user_name = "yarunachalam@suse.com";
+
+    set_var('APPLICATION_COLLECTION_USER_PASSWORD_SECRET', '<application_collection_user_token>');
+
+
+
+    # config kubectl
+    assert_script_run("mkdir -p ~/.kube");
+    assert_script_run("sudo cp /etc/rancher/rke2/rke2.yaml ~/.kube/config");
+    # assert_script_run("sudo chown $username:$username ~/.kube/config");
+    assert_script_run("kubectl config get-contexts");
+    assert_script_run("kubectl config use-context default");
+    assert_script_run("kubectl config view");
+
+    # Add cert-manager repo,install
+    assert_script_run("helm repo add $cert_repo");
+    assert_script_run("helm repo update");
+    assert_script_run("helm install cert-manager jetstack/cert-manager --namespace cert-manger --create-namespace --version v1.15.2 --set crds.enabled=true");
+
+    sleep 180;
+    assert_script_run("helm list --all-namespaces");
+    assert_script_run("kubectl get pods --all-namespaces");
+
+    # Access to Application collection registery
+    # Get docker username and password
+    assert_script_run("kubectl create ns suse-private-ai");
+    assert_script_run("kubectl create secret docker-registry application-collection --docker-server=dp.apps.rancher.io --docker-username='$docker_user_name' --docker-password='$APPLICATION_COLLECTION_USER_PASSWORD_SECRET' -n suse-private-ai", timeout => 360);
+    # Add suse-private-ai install
+    my $helm_values = get_var('HELM_CONFIG', 'https://github.com/yarunachalam/os-autoinst-distri-opensuse/tree/aistack_basic/data/aistack_values.yaml');
+    assert_script_run('ssh-keyscan -H github.com >> ~/.ssh/known_hosts', timeout => 60);
+    assert_script_run('eval $(ssh-agent -s)');
+    assert_script_run('ssh-keygen -t rsa -N "" -f ~/.ssh/id_rsa');
+    assert_script_run('chmod 600 ~/.ssh/id_rsa && ssh-add -l');
+    assert_script_run('ssh-add ~/.ssh/id_rsa');
+    assert_script_run('ssh -T git@github.com');
+
+    assert_script_run('git clone git@github.com:SUSE/private-ai-charts.git', timeout => 360);
```

Comment:
@yarunachalam the following also applies to you 

https://suse.slack.com/files/USLACKBOT/F07U43MSRCM/_action_required__review_of_changes_needed_for_common_criteria_in_the_next_few_days